Library/Homebrew/install.rb

```diff
@@ -24,12 +24,19 @@ def attempt_directory_creation
       Keg::MUST_EXIST_DIRECTORIES.each do |dir|
         begin
           FileUtils.mkdir_p(dir) unless dir.exist?
+          create_keep_files(dir: dir)
         rescue
           nil
         end
       end
     end
 
+    def create_keep_files(dir:)
+      FileUtils.touch("#{dir}/.keepme")
```

Comment:
Can move this directly into `attempt_directory_creation` as it's only called from there. A comment there (linking to the issue and mentioning 10.15) would also be nice.

Reply:
fixed